source/tyr/tyr/resources.py

```diff
@@ -550,3 +567,105 @@ def delete(self, id):
+        parser.add_argument('wheelchair', type=parser_args_type.true_false, required=False, case_sensitive=False)
+        parser.add_argument('max_walking_duration_to_pt', type=parser_args_type.float_gt_0, required=False,
+                            case_sensitive=False, help='in second')
+        parser.add_argument('max_bike_duration_to_pt', type=parser_args_type.float_gt_0, required=False,
+                            case_sensitive=False, help='in second')
+        parser.add_argument('max_bss_duration_to_pt', type=parser_args_type.float_gt_0, required=False,
+                            case_sensitive=False)
+        parser.add_argument('max_car_duration_to_pt', type=parser_args_type.float_gt_0, required=False,
+                            case_sensitive=False)
+        parser.add_argument('first_section_mode[]',
+                            type=parser_args_type.option_value(['walking', 'car', 'bss', 'bike']),
+                            required=False, action='append', case_sensitive=False, dest='first_section_mode')
+        parser.add_argument('last_section_mode[]',
+                            type=parser_args_type.option_value(['walking', 'car', 'bss', 'bike']),
+                            required=False, action='append', case_sensitive=False, dest='last_section_mode')
+        return parser.parse_args()
+
+    @marshal_with(instance_traveler_types)
+    def get(self):
+        args = self.get_args()
+        try:
+            traveler_profile = models.TravelerProfile.get_by_coverage_and_type(coverage=args.coverage,
+                                                                               traveler_type=args.traveler_type)
+            if traveler_profile is not None:
+                return traveler_profile
+            return {'error': 'traveler profile does no exist in db'}, 404
+        except (sqlalchemy.exc.IntegrityError, sqlalchemy.orm.exc.FlushError), e:
+            return {'error': str(e)}, 409
+        except Exception:
+            logging.exception("fail")
+            raise
+
+    def post(self):
+        args = self.get_args()
+        try:
+            instance = models.Instance.get_by_name(args.coverage)
+            if instance is None:
+                return {'error': "Coverage: {0} doesn't exist".format(args.coverage)}
+
+            profile = models.TravelerProfile()
+            profile.coverage_id = instance.id
+            for (attr, default_value) in default_traveler_profile_params[args.traveler_type].iteritems():
+                # override hardcoded values by args if args are not None
+                value = default_value if args.get(attr) is None else args.get(attr)
+                setattr(profile, attr, value)
+            db.session.add(profile)
+            db.session.commit()
+            return {'message': 'Profile {0} is created on {1}'.format(args.traveler_type, args.coverage)}
```

Comment:
by convention after a successful PUT or POST you have to return the created/updated object